.github/workflows/docker_img.yaml

```diff
@@ -43,6 +42,7 @@ jobs:
             - name: Scan for vulnerabilities
               uses: crazy-max/docker-scan-action@master
               with:
-                  image: connectedhomeip/chip-build${{ matrix.img }}:0.5.28
+                  # NOTE: This task validates the images built previously with latest tag
+                  image: connectedhomeip/chip-build${{ matrix.img }}:latest
```

Comment:
Is there a reason we are specifying latest and not 0.5.33 here?

Reply:
@msandstedt this action scans vulnerabilities on docker images, so this workflow is only triggered when a new change occurs on any Dockerfile which reports if the new changes (submitted in the PR) doesn't affect the building process as well as the number of vulnerabilities. Using a specific docker tag will verify same public hosted images resulting in false positives.

Reply:
Ah, got it.  Thanks.